pkg/scheduler/testing/wrappers.go

```diff
@@ -390,3 +390,18 @@ func (n *NodeWrapper) Label(k, v string) *NodeWrapper {
 	n.Labels[k] = v
 	return n
 }
+
+func (n *NodeWrapper) Unschedulable(v bool) *NodeWrapper {
+	n.Spec.Unschedulable = v
+	return n
+}
+
+func (n *NodeWrapper) Capacity(capacity v1.ResourceList) *NodeWrapper {
+	n.Status.Capacity = capacity
+	return n
+}
+
+func (n *NodeWrapper) Images(images []v1.ContainerImage) *NodeWrapper {
```

Comment:
Similar as above, this function should take the burden of building []v1.ContainerImage inside, so users just pass in image names, etc.

Reply:
+1